docs/PyCDE.md

```diff
@@ -28,13 +28,25 @@ This will create a `pycde-<version>-<python version>-<platform>.whl` file in the
 
 ## Manual Compilation
 
-If you are interested in developing PyCDE, or simply want to install it yourself with CMake, you can configure CMake similarly to the [Python bindings](/PythonBindings/#manual-compilation):
+Follow these steps to setup your repository for installing PyCDE via CMake. Make sure that your repo has the proper LLVM/MLIR Python requirements by running the following from your CIRCT repo root:
+
+```
+$ python -m pip install -r llvm/mlir/python/requirements.txt
```

Comment:
the toml requirements _should_ subsume the mlir requirements, so the mlir requirements shouldn't be necessary.

Reply:
> Is it possible to avoid duplicating the requirements into a requirements.txt? It's not a huge deal but I worry about drift. It seems like pyproject.toml can be the source of truth.

pyproject.toml actually has a superset of the requirements to build and test without building a wheel. If anything, this requirements.txt could be used to construct pyproject.toml, but that's a bigger project than I want to take on right now.